R/constructNWISURL.R

```diff
@@ -160,47 +157,45 @@ constructNWISURL <- function(siteNumbers,
            if (grepl("(?i)annual", statReportType) && (grepl("-", startDate) || grepl("-", endDate))) {
              stop("Start and end dates for annual statReportType can only include years")
            }
-           statType <- paste(statType, collapse = ",")
-           parameterCd <- paste(parameterCd, collapse = ",")
-           url <- appendDrURL(baseURL,
+
+           url <- httr2::req_url_query(baseURL,
                               sites = siteNumbers,
-                              statType = statType,
-                              statReportType = statReportType,
-                              parameterCd = parameterCd
-           )
+                              .multi = "comma")
+           url <- httr2::req_url_query(url,
+                                       statReportType = statReportType,
+                                       .multi = "comma")
+           url <- httr2::req_url_query(url, statType = statType,
+                                       .multi = "comma")
+           url <- httr2::req_url_query(url, parameterCd = parameterCd, 
+                                       .multi = "comma")
+                                       
            if (nzchar(startDate)) {
-             url <- appendDrURL(url, startDT = startDate)
+             url <- httr2::req_url_query(url, startDT = startDate)
            }
            if (nzchar(endDate)) {
-             url <- appendDrURL(url, endDT = endDate)
+             url <- httr2::req_url_query(url, endDT = endDate)
            }
            if (!grepl("(?i)daily", statReportType)) {
-             url <- appendDrURL(url, missingData = "off")
+             url <- httr2::req_url_query(url, missingData = "off")
            }
          },
          gwlevels = {
            
-           url <- appendDrURL(baseURL,
-                              site_no = siteNumbers,
-                              format = "rdb"
-           )
+           url <- httr2::req_url_query(baseURL,
+                              site_no = siteNumbers,.multi = "comma")
```

Comment:
```suggestion
                              site_no = siteNumbers, .multi = "comma")
```
Sorry, this is such a small nit but I saw it 😆